pom.xml

```diff
@@ -86,11 +86,5 @@
             <version>2.8.1</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
```

Comment:
Now you removed the dependency entirely, which will lead to compile errors.
Use[ this dependency](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/browse/pom.xml#80) instead.

Reply:
Also maybe i would add the message sent by the player to the event